Services/AzineHybridAuth.php

```diff
@@ -53,23 +58,23 @@ class AzineHybridAuth {
 	/**
 	 *
 	 * @param UrlGeneratorInterface $router
+	 * @param UserInterface $user
 	 * @param TokenStorageInterface $tokenStorage
 	 * @param ObjectManager $manager
 	 * @param array $config
 	 * @param bool $storeForUser
 	 * @param $storeAsCookie
+	 * @param $expiresInDays
 	 */
-	public function __construct(UrlGeneratorInterface $router, TokenStorageInterface $tokenStorage, ObjectManager $manager, $config, $storeForUser, $storeAsCookie){
+	public function __construct(UrlGeneratorInterface $router, UserInterface $user, ObjectManager $manager, $config, $storeForUser, $storeAsCookie, $expiresInDays){
```

Comment:
Why does this need to be changed? If possible, the public function signature should stay the same, so users don't have problems with BC breaks.